packages/pagination/stories/3-CursorPagination.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react';
+import { CursorPagination } from '@zendeskgarden/react-pagination';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Pagination/CursorPagination',
+  component: CursorPagination
+} as Meta;
+
+export const Default: Story = () => {
+  const [cursor, setCursor] = useState(0);
+
+  const pages = [0, 1, 2, 3, 4];
```

Comment:
Not seeing where this array is leveraged. Could it just be `const pageLength = 4`?

Reply:
Updated, thanks!

Reply:
Updated, thanks!